specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/HybridLinks.json

```diff
@@ -0,0 +1,460 @@
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create hybrid link.": {
+            "$ref": "./examples/HybridLinkCreate.json"
+          },
+          "Update hybrid link replication mode before deleting it.": {
+            "$ref": "./examples/HybridLinkUpdate.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
>delete [](start = 7, length = 6)

Please add PATCH atleast for the tags field.

Reply:
I don't need patch approach in my feature.